src/applications/mhv-supply-reordering/sass/mhv-supply-reordering.scss

```diff
@@ -4,3 +4,8 @@
 @import "~@department-of-veterans-affairs/css-library/dist/stylesheets/modules/m-modal";
 @import "~@department-of-veterans-affairs/css-library/dist/stylesheets/modules/m-omb-info";
 @import "../../../platform/forms/sass/m-form-confirmation";
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
+.mhv-c-reorder-unavail-card {
+    max-width: 486px;
```

Comment:
I know design is still being worked on, so just noting that this seems like an arbitrary value. Unsure if there are VADS variables that could get the intended effect